frontstage/templates/secure-messages/conversation-list.html

```diff
@@ -54,11 +54,13 @@
                 {
                   "id": "message-link-" ~ loop.index,
                   "unread": true if message['unread'] else false,
-                  "url": url_for('secure_message_bp.view_conversation', thread_id=message.thread_id) ~ "#latest-message",
-                  "subject": message.subject,
+                  "subject": {
+                                "url": url_for('secure_message_bp.view_conversation', thread_id=message.thread_id) ~ "#latest-message",
+                                "text": message.subject
+                                },
                   "fromText": message.from or 'Unavailable',
                   "dateText": message.sent_date or 'Unavailable',
-                  "body": '<xmp>'+ message['body']|truncate(80, False, '...', 0) +'</xmp>'
+                  "body": '<pre>'+ message['body']|truncate(80, False, '...', 0) +'</pre>'
```

Comment:
The `xmp` tag was deprecated in html 3.2 (released in 1996). so unless anyone can seen any issues and is particularly attached to this tag it should be replaced with `pre` instead